### ISSUE TITLE
Create a media library uploader form component

### ISSUE THREAD

**Issue by JanLenoch** [Kentico/xperience-training-13#67]
### Who

- [x] Anonymous visitor
- [ ] Patient
- [ ] Doctor
- [ ] Content editor
- [ ] Marketer
- [ ] Administrator
- [ ] Developer

### Needs what

To upload files into a dedicated media library.

### Why

Why is the feature needed?

### Issue expands upon

This issue expands upon another issue(s): #`issue number`

### Blockers

This issue is blocked by the following issue(s): #`issue number`

**Comment by JanLenoch** [comment]
Resolved via 97cc8ce4c300614a3cb53a2df1a53f289213e844
